addons/mail/static/src/utils/messaging_component.js

```diff
@@ -33,10 +35,34 @@ export function registerMessagingComponent(ComponentClass, { propsCompareDepth =
             // useModels as if they were part of the OWL rendering itself.
             useModels();
             useShouldUpdateBasedOnProps({ propsCompareDepth });
+            onMounted(() => {
+                for (const className of this.classList) {
+                    this.el.classList.add(className);
+                }
+            });
+            onPatched(() => {
+                for (const className of this.classList) {
+                    this.el.classList.add(className);
+                }
+            });
```

Comment:
For current use it doesn't matter, but if we want something completely generic it should handle removing classes that are no longer useful.

As I have posted on another PR, we might be better off finding a way to automatically pre-process the template to add them on it automatically (instead of reimplementing part of the logic that OWl already handles for several things).